momento-sdk/src/main/java/momento/sdk/ScsDataClient.java

```diff
@@ -486,19 +486,19 @@ private _DeleteRequest buildDeleteRequest(ByteString key) {
     return _DeleteRequest.newBuilder().setCacheKey(key).build();
   }
 
-  private _SetRequest buildSetRequest(ByteString key, ByteString value, long ttl) {
+  private _SetRequest buildSetRequest(ByteString key, ByteString value, Duration ttl) {
     return _SetRequest.newBuilder()
         .setCacheKey(key)
         .setCacheBody(value)
-        .setTtlMilliseconds(ttl)
+        .setTtlMilliseconds(ttl.getSeconds() * 1000)
```

Comment:
just curious - does the Duration object not have a `getMilliseconds()`?

Reply:
Looks like it has a `toMillis()` method.